apps/api/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     environment:
       NODE_ENV: production
 
-  mariadb:
+  osmox-mariadb:
```

Comment:
What about portal docker?

Also can we add image name for osmox-api as the same?
![image](https://github.com/OsmosysSoftware/osmo-x/assets/16570290/6749f5f8-3e9a-4588-8a91-7ebba269cc36)

Reply:
Can we add it for portal too?

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/234/commits/86e5ecc1fab975b75a95917cd79593a5af53a2fa